### PR TITLE
Fix sequence diagram of update workload

### PR DESCRIPTION
**Pull request by inf17101**
Introduced with #1 the Agent Manager does only receive the ExecutionCommands of the server, but forwards the actions to RuntimeManager. Thus the requirement and the sequence diagram needs an update.

I have made the message "update workload" to the Runtime Manager from the Agent Manager async to decouple at this time point, even if it is sequentially because of the await. The real decoupling happens later in the Runtime Facade but this is not part of this diagram. But at least with the async arrow it behaves correctly that the Agent Manger is not blocked.

# Definition of Done

The PR shall be merged only if all items mentioned in [CONTRIBUTING.md](https://github.com/eclipse-ankaios/ankaios/blob/main/CONTRIBUTING.md#how-to-contribute) have been followed. In case an item is not applicable as described, please provide a short explanation in the description.

- [x] All steps in [CONTRIBUTING.md](https://github.com/eclipse-ankaios/ankaios/blob/main/CONTRIBUTING.md#how-to-contribute) have been handled

### CODE REVIEW

File: agent/doc/swdesign/README.md [hunk 1]
```diff
@@ -254,15 +254,15 @@ The following diagram show the general steps the Ankaios Agent takes when receiv
...
 ![Handling UpdateWorkload](plantuml/seq_update_workload.svg)
 
-#### Agent Manager creates Control Interface Instance for each workload
+#### RuntimeManager creates Control Interface Instance for each workload
 `swdd~agent-create-control-interface-pipes-per-workload~1`
 
 Status: approved
 
-When the Agent Manager gets an `UpdateWorkload` message, for each added Workload with a runtime that is known, the Agent Manager shall create a Control Interface Instance for this Workload.
+When the AgentManager gets an `UpdateWorkload` message, for each added Workload with a runtime that is known, the RuntimeManager shall create a Control Interface Instance for this Workload.
```
Comment (krucod3) [inline agent/doc/swdesign/README.md:262]:
```suggestion
When the RuntimeManager receives a call to handle an `UpdateWorkload` message, for each added Workload with a runtime that is known, the RuntimeManager shall create a Control Interface Instance for this Workload.
```

**Review by krucod3 — COMMENTED** [review]
Looks good, just a small suggestion.

**Comment by inf17101** [comment]
> Looks good, just a small suggestion.

I have updated with your suggestion. Thank you.

**Review by krucod3 — APPROVED** [review]
LGTM 👍

**Pull request merged**